docs/app/content-collections.md

```diff
@@ -1,168 +1,113 @@
-3. Click "Layout Options" in the page sidebar
-4. Choose the desired "Layout" you want to use
-5. Configure any other layout options
-
-### Table Layout
-
-Supports all forms of data, and is therefore the default within the Content module. It includes the following
-features/options:
-
-- Toggle & Reorder Field Columns (Option)
-- Row Spacing (Option)
-- Manual Item Ordering
-- Column Sort
-- Column Resizing
-- Select All
+::: tip
 
-### Card Layout
-
-This tiled layout is ideal for collections that prioritize an image, and is the default for both the
-[User Directory](/app/user-directory/) and [File Library](/reference/files/). It includes the following
-features/options:
-
-- Image Source (Option)
-- Title (Option)
-- Subtitle (Option)
-- Image Fit (Option)
-- Fallback Icon (Option)
-- Card Size
-- Sort Field & Direction
-- Select All
-
-### Calendar Layout
-
-An excellent choice for temporal (date/time) data, this Layout allows you to choose between a Month, Week, Day, and List
-view. The following options are included:
-
-- Display Template
-- Start Date Field
-- End Date Field
+The new sort order is saved automatically, so make sure you wait until the App's loading indicator finishes before
+navigating away from the page.
 
-### Map Layout
+:::
 
-For geospatial/geographic data, the Map Layout provides a world map for points, lines, and other geometry. The following
-options are available:
+:::tip
 
-- Basemap
-- Location Field
-- Display Template
-- Enable Clustering
+Sorting saves a value in a [Sort Field](/reference/query/#sort). This value is used to return data in a custom order.
+The key takeaway is that while the values in this Sort Field do get updated and modified, values in other Fields are not
+changed.
 
-## Searching Items
+:::
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to search
-3. Type your search query into the input (magnifying glass icon) located in the header
+## Order Items
 
-## Filtering Items
+![Order Items](image.webp)
 
-**[Learn more about Filtering](/configuration/filter-rules/).**
+Order and browse Items alphabetically or numerically. Ordering Items is not possible on certain Layouts, such as the Map
+Layout. To order a Collection's Items, follow the steps below.
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to filter
-3. Click on Filter (filter icon) located in the header
-4. Click the "Add Filter" dropdown and select the field to filter on
-5. Adjust the operator (eg: "Contains") as needed by clicking on it
-6. Enter a value in the field filter's input(s)
-7. Remove unwanted filters by hovering over the field and clicking "X"
+1. Navigate to the Content Module and select the desired Collection.
+2. In the Sidebar, choose a [Layout](/app/content-layouts/#adjust-item-layouts) that supports ordering.\
+   (e.g. "table" or "card" mode)
```

Comment:
Do we like this visual breakup? In some of the previous docs I put the parenthesis comments on the same line.
I think I personally like this style more.